sdktests/testsuite_entry_point.go

```diff
@@ -70,7 +70,6 @@ func RunSDKTestSuite(
 }
 
 func doAllServerSideTests(t *ldtest.T) {
-	t.Run("data store", doServerSideDataStoreTests)
 	t.Run("evaluation", doServerSideEvalTests)
```

Comment:
I think I missed this - so have these tests been moved into one of the common files?

Reply:
Sorry, I forgot to comment on that: they are moved and renamed, so they're now the "updates" part of the "stream" tests. They are really testing both the stream update functionality _and_ the update-versioning functionality that's part of the in-memory data store, but there's not really a way to test the latter without the former, and any failures are much more likely to be from the former, so I figured it was clearer to call them stream tests.

Reply:
See:
* https://github.com/launchdarkly/sdk-test-harness/pull/77/files#diff-c366be2f3b60d321e72c92cf2eb34a673ee62965ee6eb6293146bcec3a671515R11
* https://github.com/launchdarkly/sdk-test-harness/pull/77/files#diff-3fd8a54b081125b9520ab25675cb065e143000abd9da7131129a99e25fc4507fR13
* https://github.com/launchdarkly/sdk-test-harness/pull/77/files#diff-6b419db42cb6d705a3cd03fc68e8e5f209dcd3aa944d48edb61ede2a518e7cbb